CERT-Polska/malwarecage
# Complete API documentation

**Your checklist for this pull request**

**What is the current behaviour?**
- API documentation is inconsistent, contains few errors and has undocumented methods that should be documented

![image](https://user-images.githubusercontent.com/8720367/84898585-d18e1c80-b0a7-11ea-8529-8ae817c15143.png)


**What is the new behaviour?**
- Bumped apispec to latest to fix some issues (e.g. https://github.com/marshmallow-code/apispec/issues/209)
- API documentation with much better coverage, accessible via `/docs` web endpoint.

![image](https://user-images.githubusercontent.com/8720367/84898663-e8347380-b0a7-11ea-94d8-aa5d53885b06.png)


**Test plan**
- Go to the `/docs` and check the completion and readability.
- Expand all the endpoints in Swagger to test for errors (components are lazy-loaded)
- Check "try it out" functionality
